testproject/Assets/Tests/Runtime/MultiprocessRuntime/ExecuteStepInContext.cs

```diff
@@ -56,7 +56,7 @@ public IEnumerator AfterTest(ITest test)
     private bool m_IgnoreTimeoutException;
 
     private float m_StartTime;
-    private bool isTimingOut => Time.time - m_StartTime > TestCoordinator.MaxWaitTimeoutSec;
+    private bool isTimingOut => m_StartTime < Time.time;
```

Comment:
won't this always return true?

Reply:
oh just saw the line below. we might want to rename this var now, as it's not the "start time" anymore

Reply:
m_Timeout

Reply:
renamed variable